packages/client/src/app/pages/portfolio/my-stuff/my-stuff.component.html

```diff
@@ -0,0 +1,185 @@
+                <i-feather class="no-text" name="edit-3"></i-feather>
+            </button>
+            <button matRipple class="tool end disabled" matTooltip="Delete" matTooltipClass="offprint-tooltip">
+                <i-feather class="no-text" name="trash-2"></i-feather>
+            </button>
+            <span class="tool-divider"></span>
+            <button matRipple class="tool disabled">
+                <i-feather name="book"></i-feather>Unpublished
+            </button>
+        </div>
+        <form [formGroup]="searchStuff" class="tool-search">
+            <div class="search-bar">
+                <input type="text" formControlName="query" class="search-input disabled" placeholder="Search your stuff..." disabled>
+                <button matRipple class="search-submit disabled" type="submit"><i-feather name="search"></i-feather></button>
+            </div>
+        </form>
+    </ng-template>
+</div>
+
+<mat-menu #createMenu="matMenu">
+    <button mat-menu-item [routerLink]="['new-work']"><i-feather name="book-open"></i-feather>Add Fiction</button>
+    <button mat-menu-item><i-feather name="feather"></i-feather>Add Poetry</button>
+    <button mat-menu-item><i-feather name="globe"></i-feather>Add Nonfiction</button>
+    <button mat-menu-item [routerLink]="['new-blog']"><i-feather name="coffee"></i-feather>Add Blog</button>
+</mat-menu>
+
+<mat-menu #filterMenu="matMenu">
+    <button mat-menu-item>By Name</button>
+    <button mat-menu-item>By Kind</button>
+    <button mat-menu-item>By Date Added</button>
+</mat-menu>
+
+<!--<div class="folder-routing">
+    <ng-container *ngIf="currFolder; else topLevel">
+        <h3><a (click)="goHome()">Home</a></h3> <i-feather name="chevron-right"></i-feather>
+        <ng-container *ngFor="let parent of $any(currFolder.parents)">
+            <h3>{{ parent.name }}</h3> <i-feather name="chevron-right"></i-feather>
+        </ng-container>
+        <h3 class="active">{{ currFolder.name }}</h3>
+    </ng-container>
+    <ng-template #topLevel>
+        <h3 class="active">Home</h3>
+    </ng-template>
+</div>-->
+
+<ng-container *ngIf="route.children.length === 0; else openedForms">
+    <!--User nas nothing-->
+    <ng-container *ngIf="myContent.length === 0; else hasContent">
+        <div class="empty">
+            <h3>It's pretty lonely here...</h3>
+            <p>Why not add some content by hitting the create button?</p>
+        </div>
+    </ng-container>
+        
+    <!--User has content-->
+    <ng-template #hasContent>
+        <div class="content-grid">
+            <ng-container *ngIf="myContent.length === 0; else hasContentTop">
+                <div class="empty" style="width: 100%;">
+                    <h3>It's pretty lonely here...</h3>
+                    <p>Why not add some content by hitting the create button?</p>
+                </div>
+            </ng-container>
```

Comment:
What's the distinction between these two? It seems like they're distinct forms of UI that are triggered by the same condition.